src/webpack/config.js

```diff
@@ -107,14 +136,21 @@ export function webpackConfig(env) {
 
   if (isProd) {
     // fix loaders for prod
-    const [, cssLoader] = loaders;
+    const [, thirdPartyCssLoader, cssLoader] = loaders;
```

Comment:
I don't follow all this, but where is the third party one split from the regular loader?